### PR TITLE
you can now flip eyepatches + blindfolds mix eye colors

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request
you can flip eyepatches using them inhand (or an action button)
white blindfolds now blend your eye colors, like the comment says it should

## Why It's Good For The Game
not a great eyepatch if it only works for one eye.

## Changelog
:cl:
add: you can flip eyepatches
fix: blindfolds now take into consideration both eye colors for their color
/:cl:

### CODE REVIEW

**Review by Imaginos16 — APPROVED** [review]
Fucking finally.

**Comment by CMDR-Gungnir** [comment]
FINALLY

**Comment by Overuseddart** [comment]
Let me wear two eyepatches PLEASE

**Comment by Twaticus** [comment]
huge

File: code/modules/clothing/glasses/_glasses.dm [hunk 1]
```diff
@@ -183,6 +183,12 @@
 	desc = "Yarr."
 	icon_state = "eyepatch"
 	inhand_icon_state = null
+	actions_types = list(/datum/action/item_action/toggle)
+
+/obj/item/clothing/glasses/eyepatch/attack_self(mob/user, modifiers)
+	. = ..()
+	icon_state = icon_state == "eyepatch_flipped" ? "eyepatch" : "eyepatch_flipped"
```
Comment (Watermelon914) [inline code/modules/clothing/glasses/_glasses.dm:190]: this really shouldn't be hardcoded. Use a variable for the flipped and non-flipped states

File: code/modules/clothing/glasses/_glasses.dm [hunk 2]
```diff
@@ -505,6 +511,12 @@
 	desc = "An eyepatch with built-in thermal optics."
 	icon_state = "eyepatch"
 	inhand_icon_state = null
+	actions_types = list(/datum/action/item_action/toggle)
+
+/obj/item/clothing/glasses/thermal/eyepatch/attack_self(mob/user, modifiers)
+	. = ..()
+	icon_state = icon_state == "eyepatch_flipped" ? "eyepatch" : "eyepatch_flipped"
```
Comment (Watermelon914) [inline code/modules/clothing/glasses/_glasses.dm:518]: same thing here

File: code/modules/clothing/glasses/hud.dm [hunk 3]
```diff
@@ -151,6 +151,12 @@
 	name = "eyepatch HUD"
 	desc = "The cooler looking cousin of HUDSunglasses."
 	icon_state = "hudpatch"
+	actions_types = list(/datum/action/item_action/toggle)
+
+/obj/item/clothing/glasses/hud/security/sunglasses/eyepatch/attack_self(mob/user, modifiers)
+	. = ..()
+	icon_state = icon_state == "hudpatch_flipped" ? "hudpatch" : "hudpatch_flipped"
```
Comment (Watermelon914) [inline code/modules/clothing/glasses/hud.dm:158]: and here

**Review by Watermelon914 — CHANGES_REQUESTED** [review]

**Pull request merged**